src/shared/util.js

```diff
@@ -1679,7 +1682,7 @@ function loadJpegStream(id, imageUrl, objs) {
   img.src = imageUrl;
 }
 
-//#if !(MOZCENTRAL)
+if (typeof PDFJSDev === 'undefined' || !PDFJSDev.test('MOZCENTRAL')) {
 //// Polyfill from https://github.com/Polymer/URL
```

Comment:
Nit: with the new proprocessor we shouldn't need double comments here, so `////` can be changed to `//`.